examples/data_sharing.tex

```diff
@@ -0,0 +1,211 @@
+
+\section{Subprocessing}
+
+\subsection{Processor shall not appoint (or disclose any \gls{type} Personal Data to) any Subprocessor unless required or authorized by \gls{undertaking}.}
+
+\section{Data Subject Rights}
+
+\subsection{Taking into account the nature of the Processing, \gls{processor} shall assist \gls{undertaking} by implementing appropriate technical and organisational measures, insofar as this is possible, for the fulfilment of \gls{undertaking} requests to exercise Data Subject rights under the Data Protection Laws.}
+
+\subsection{Processor shall}
+
+\subsubsection{promptly notify \gls{undertaking} if it receives a request from a Data Subject under any Data Protection Law in respect of \gls{undertaking} Personal Data;}
+
+\subsection*{ensure that it does not respond to that request except on the documented instructions of \gls{undertaking} or as required by Applicable Laws to which \gls{processor} is subject, in which case \gls{processor} shall to the extent permitted by Applicable Laws inform Company of that legal requirement before the Contracted Processor responds to the request.}
+
+\section{Personal Data Breach}
+
+\subsection{\Gls{processor} shall notify \glsfmttext{undertaking} without undue delay upon \glsfmttext{processor} becoming aware of a Personal Data Breach affecting \glsfmttext{undertaking} Personal Data, providing \glsfmttext{undertaking} with sufficient information to allow \glsfmttext{undertaking} to meet any obligations to report or inform Data Subjects of the Personal Data Breach under the Data Protection Laws. }
+
+\Glsfmttext{processor} shall co-operate with the \glsfmttext{undertaking} and take reasonable commercial steps as are directed by \glsfmttext{undertaking} to assist in the investigation, mitigation and remediation of each such Personal Data Breach.
+
+\section{Data Protection Impact Assessment and Prior Consultation}
+
+Data Protection Impact Assessment and Prior Consultation Processor shall provide reasonable assistance to \gls{undertaking} with any data protection impact assessments, and prior consultations with Supervising Authorities or other competent data privacy authorities, which \gls{undertaking} reasonably considers to be required by article 35 or 36 of the GDPR or equivalent provisions of any other Data Protection Law, in each case solely in relation to Processing of \gls{type} Personal Data by, and taking into account the nature of the Processing and information available to, the Contracted Processors.
+
+\section{Deletion or return of \glsfmttext{type} Personal Data}
+\label{deletion}
+
+\subsection{Subject to this section \ref{deletion} \glsfmttext{processor} shall promptly and in any event within 10 business days of the date of cessation of any Services involving the Processing of \glsfmttext{type} Personal Data (the ``\textbf{Cessation Date}''), delete and procure the deletion of all copies of those \glsfmttext{undertaking} Personal Data.}
```

Comment:
s/section \ref/section~\ref/g

Reply:
some clarity to this comment would be nice. While I agree with using `~` instead of ` `, the rest of this suggestion IMO decreases readability, not desired.

Reply:
what's "the rest of this suggestion" referring to?

Reply:
`s/section \ref/section` and `\ref/g`. Not quite sure what that all does/means.

Reply:
How about just 'autoref', which will print 'section' (if it's a section) and 'subsection' (if it's a subsection), &c?

Reply:
@bond7o6 oh sorry. `s/section \ref/section~\ref/g` means "substitute `section \ref` with `section~\ref` globally / in all occurences" in sed. 

@Andonome in that case i'd suggest `cleveref` over `autoref`. wdyt?

Reply:
Oh, is that what they're on about? Yea, switch it to autoref and we'll discuss it in depth later.

Reply:
I've never used 'cleveref', so I can't compare. I'm sure either's fine, but if you use 'cleveref', you should also remove the 'hyperref' package, as that's all I'm using it for.

Reply:
ahhhh, my reliance on emacs is showing itself there). I 'think' hyperref is the preferred choice over cleverref, and autoref is a command within hyperref, not another package.

Reply:
also moved this to #38 because it can be handled outside of the PR